src/config.js

```diff
@@ -12,8 +12,9 @@ const Config = {
             'X-Requested-With, content-type, x-access-token'
         );
         res.header('Access-Control-Allow-Credentials', true);
-
-        console.log(`${req.ip} ${req.method} ${req.url}`);
+        if (process.env.PROD) {
+            console.log(`${req.ip} ${req.method} ${req.url}`);
+        }
```

Comment:
Exactly! In development the request IP will always be the same and the endpoint (i.e. method + url) targeted will probably be known. In production this could be used to debug requests from unknown clients.